### PR TITLE
Feat/image input

### PR DESCRIPTION
**Pull request by bhavyaj12**
Added image and input components

- Image has 2 variants, responsive and round image
- Input has 3 variants. Form with validation styles of both error and success are present. JS is not included, will add it later as for now only styles is necessary.
- Haven't completed making everything responsive yet

link: https://flamboyant-spence-630d96.netlify.app/

### CODE REVIEW

**Review by Asuraking1n — COMMENTED** [review]
Everything looks good to me in desktop and tab view, but mobile view content overlaps, I guess it not responsive yet

File: pages/trial-code.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        <h3 class="subheader-component txt-medium">Trial Code</h3>
+        <p class="txt-component txt-normal">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+            ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+            velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum.</p>
+
+        <div class="component-example">
+
+        </div>
```
Comment (arjunbharti) [inline pages/trial-code.html:76]: If it's not required then you can delete it.

**Comment by arjunbharti** [comment]
You can remove extra spaces in the code. Added some comments also.

File: pages/trial-code.html (hunk 1 above)
Comment (bhavyaj12) [reply pages/trial-code.html:76]: Yes I will, thanks a lot!

**Pull request merged**